sustainability/src/main/java/edu/asu/diging/sustainability/core/model/impl/Concept.java

```diff
@@ -81,6 +82,8 @@ public void setParent(IConcept parent) {
 
     @Override
     public List<IConcept> getChildren() {
+        Comparator<IConcept> byName = Comparator.comparing(IConcept::getName);
+        children.sort(byName);
```

Comment:
I'm sorting the child concepts here, these would be the SDGs if I'm not wrong. I've tested this code and it sorts both the concepts as well as the child concepts. Is there something that I'm missing?